chaco/tools/toolbars/plot_toolbar.py

```diff
@@ -282,12 +281,12 @@ def _calculate_height(self):
             self._layout_needed = True
             self.request_redraw()
 
-    @on_trait_change('hiding')
-    def _hiding_changed(self):
+    @observe('hiding')
+    def _on_hiding_changed(self, event):
         self._layout_needed = True
         self.request_redraw()
 
-    @on_trait_change('auto_hide')
-    def _auto_hide_changed(self):
+    @observe('auto_hide')
+    def _on_auto_hide_changed(self, event):
```

Comment:
I wish we could come up with better names for `_on_hiding_changed` and `_on_auto_hide_changed`.

Maybe `_set_layout_needed_and_redraw` and `_set_hiding_and_redraw`?

The names sound kinda silly now because the code in the methods almost matches up with the method names.